frappe/utils/data.py

```diff
@@ -1551,15 +1551,15 @@ def pretty_date(iso_datetime: datetime.datetime | str) -> str:
 		return _("Yesterday")
 	elif dt_diff_days < 7.0:
 		return _("{0} days ago").format(cint(dt_diff_days))
-	elif dt_diff_days < 12:
+	elif dt_diff_days < 14:
```

Comment:
Cut offs are not exact because it's somewhat more user friendly.


I know it sounds weird. But nearly 2 weeks shouldn't be "1 week away"

So maybe leave cutoff times same but handle the case in plular branch of the code.

Reply:
does it look good now?